### PR TITLE
Fix #285

### LINKED ISSUE #285: [app-grid] Wrong math in "--app-grid-expandible-item"?
In an grid with this characteristics:

width = 480
gutter = 16
columns = 4

...the results should be:
```
1-1-1-1 = 16 [100] 16 [100] 16 [100] 16 [100] 16 = 480
    2-2 = 16 [-----216----] 16 [-----216----] 16 = 480
    3-1 = 16 [---------332---------] 16 [100] 16 = 480
      4 = 16 [--------------448-------------] 16 = 480
```

1 column = 100
2 columns = 216
3 columns = 332
4 columns = 448

...but the math in "--app-grid-expandible-item" is generating this results:

1 column = 68
2 columns = 200
3 columns = 332
4 columns = 448

Am I missing something?

### PR DESCRIPTION
**Pull request by blasten**
Fixes the bug and the demo shows the photos I took in Sydney!

### CODE REVIEW

File: app-grid/demo/flickr-grid-layout.html [hunk 1]
```diff
@@ -66,6 +66,10 @@
           list-style: none;
         }
 
+        [title] {
```
Comment (frankiefu) [inline app-grid/demo/flickr-grid-layout.html]: Should this be `main-title` instead?

**Comment by frankiefu** [comment]
Awesome pictures!!!

File: app-grid/demo/flickr-grid-layout.html (hunk 1 above)
Comment (blasten) [inline app-grid/demo/flickr-grid-layout.html]: 👍

**Comment by blasten** [comment]
Now, I would have to upload the +300 photos. Not very productive!

**Comment by frankiefu** [comment]
LGTM

**Pull request merged**